src/ActionModal/components/ActionModal.vue

```diff
@@ -942,7 +942,6 @@ export default {
       variables() {
         let { type, ...message } = this.transactionData
         delete message.memo
-        delete message.addressRole
```

Comment:
why the delete?

Reply:
We need to pass addressRole to API to be able to calculate the fee in a stakeTx.